pkg/framework/objects/service_token/resource_acceptance_test.go

```diff
@@ -1,4 +1,4 @@
-package resources_test
+package service_token_test
```

Comment:
Can we add a test or update the existing one where we add some environment level restrictions?

Reply:
I will add. Can we check to make sure that the feature is enabled in the instance used by CI ?

Reply:
It is, because I implemented the env level for groups already and added tests for it.

The same flag is used for groups and service tokens.